app/assets/javascripts/course/experience_points_disbursement.js

```diff
@@ -0,0 +1,12 @@
+(function($) {
+  'use strict';
+  var COPY_BUTTON_SELECTOR = '#experience-points-disbursement-copy-button';
+
+  function copyExperiencePoints(event) {
+    event.preventDefault();
+    var valueToCopy = $('.course_user:first').find('.points_awarded:first').val();
+    $('.points_awarded').val(valueToCopy);
+  }
+
+  $(document).on('click', COPY_BUTTON_SELECTOR, copyExperiencePoints);
```

Comment:
@allenwq @kxmbrian you always need to qualify the selector with the controller and action CSS classes.